nuxt.config.js

```diff
@@ -101,7 +101,7 @@ export default {
    */
   publicRuntimeConfig: {
     avroApiBaseUrl:
-      process.env.AVRO_API_BASE_URL || 'https://avro.alerce.online',
+      process.env.AVRO_API_BASE_URL || 'https://dev.avro.alerce.online',
```

Comment:
I thought we were ready with the production version, why keep the staging?

Reply:
The default is fine with staging env. The deployment action uses a secret as environment variable with the correct URL

Reply:
Just noticed it's only the default value